src/action-menu/ActionMenu.module.scss

```diff
@@ -0,0 +1,163 @@
+// @import '../themes/clinical-lowlight-theme/ActionMenu.module';
+// @import '../themes/orion-fusion-theme/ActionMenu.module';
+
+:local {
+  .action-menu,
```

Comment:
why do you put all the styles in a single file instead of splitting it up per component?

Reply:
Updated.